doc/userdoc/guides/nest2_to_nest3/nest2_to_nest3_overview.rst

```diff
@@ -24,6 +24,19 @@ perform operations that were not possible in previous versions.
 What's new?
 -----------
 
+NEST Server
```

Comment:
This  file is removed in #1929 and since the info is pretty much what the nest-server doc says I think it could be removed. I can add link to the nest-server doc from the new features of nest3 in that PR.